st2-ask/notification_rule.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: "notify_slack_execution_change"
+pack: "st2"
+description: "Watches for an execution status of "awaits" for a certain action ref
+enabled: true
+
+trigger:
+  type: core.st2.generic.actiontrigger
+
+criteria:
+  trigger.action_ref:
+      type: equals
+      pattern : st2.ask
+  trigger.status:
+      type: equals
+      pattern : awaits
```

Comment:
Fine by me, think this was just a placeholder from @enykeev original suggestion. Don't think anyone is particularly married to `awaits`